src/main/java/com/dreammaster/gthandler/GT_Loader_Materials.java

```diff
@@ -33,23 +33,23 @@ private void registerMaterialRecipes()
 
 		GT_ModHandler.addShapelessCraftingRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Duralumin, 8L),
 				GT_ModHandler.RecipeBits.NOT_REMOVABLE, new Object[] {
-						OrePrefixes.dust.get(Materials.Aluminium),
-						OrePrefixes.dust.get(Materials.Aluminium),
-						OrePrefixes.dust.get(Materials.Aluminium),
-						OrePrefixes.dust.get(Materials.Aluminium),
-						OrePrefixes.dust.get(Materials.Aluminium),
-						OrePrefixes.dust.get(Materials.Aluminium),
+						OrePrefixes.dust.get(Materials.Aluminiumoxide),
+						OrePrefixes.dust.get(Materials.Aluminiumoxide),
+						OrePrefixes.dust.get(Materials.Aluminiumoxide),
+						OrePrefixes.dust.get(Materials.Aluminiumoxide),
+						OrePrefixes.dust.get(Materials.Aluminiumoxide),
+						OrePrefixes.dust.get(Materials.Aluminiumoxide),
 						OrePrefixes.dust.get(Materials.AnyCopper),
 						OrePrefixes.dust.get(Materials.Manganese),
 						OrePrefixes.dust.get(Materials.Magnesium)});
 
 		GT_ModHandler.addShapelessCraftingRecipe(GT_ModHandler.getModItem("dreamcraft", "item.AlumiteDust", 9L),
 				GT_ModHandler.RecipeBits.NOT_REMOVABLE, new Object[]{
-						OrePrefixes.dust.get(Materials.Aluminium),
-						OrePrefixes.dust.get(Materials.Aluminium),
-						OrePrefixes.dust.get(Materials.Aluminium),
-						OrePrefixes.dust.get(Materials.Aluminium),
-						OrePrefixes.dust.get(Materials.Aluminium),
+						OrePrefixes.dust.get(Materials.Aluminiumoxide),
```

Comment:
Aren't these two alloying recipes? Should they be changed as well?

Reply:
changed

Reply:
there are also some alloy that you changed the silicon and alu in gt5's pr

Reply:
@GlodBlock can you link it in gt5 pr ?